d3d/HLSL_ShaderModel6_8.md

```diff
@@ -1,35 +1,43 @@
 <h1>HLSL Shader Model 6.8</h1>
 
-v0.5 2023-06-21
+v0.6 2024-03-11
 
-<h2> Contents</h2>
+<h2>Contents</h2>
 
 - [Introduction](#introduction)
-- [Work Graphs (experimental)](#work-graphs-experimental)
-- [Wave Matrix (experimental)](#wave-matrix-experimental)
+- [Expanded Comparison Sampling](#expanded-comparison-sampling)
+- [Wave Size Range](#wave-size-range)
+- [Work Graphs](#work-graphs)
 - [Change Log](#change-log)
 
-# Introduction
+## Introduction
 
 This document covers the new Shader Model 6.8.  A brief summary of each new feature
 is listed below along with links to detailed specifications.
 
-# Work Graphs (experimental)
+## Expanded Comparison Sampling
 
-Work Graphs define a system of shader nodes that feed into each other
-to enable tailored GPU work creation.
+Several existing operations have been expanded to support Comparison Samplers.
+New methods: `SampleCmpBias`, `SampleCmpGrad`.
+Added overloads: `CalculateLevelOfDetail`, `CalculateLevelOfDetailUnclamped`.
+
+See the [Expanded Comparison Sampling](https://microsoft.github.io/hlsl-specs/proposals/0014-expanded-comparison-sampling.html) documentation for more details.
+
+## Wave Size Range
 
-See the [Work Graphs (experimental)](WorkGraphs.md) documentation for more details.
+A new variant of the [WaveSize](HLSL_SM_6_6_WaveSize.md) attribute is added: `[WaveSize( min, max [, preferred] )]`.
+This allows you to specify a range and optional preferred size.
 
-# Wave Matrix (experimental)
+See the [Wave Size Range](https://microsoft.github.io/hlsl-specs/proposals/0013-wave-size-range.html) documentation for more details.
 
-Wave Matrices are type abstractions of hardware support
-for higher bandwidth matrix multiplications, useful
-in machine learning and image processing.
+## Work Graphs
+
+Work Graphs define a system of shader nodes that feed into each other
+to enable tailored GPU work creation.
 
-See the [Wave Matrix (experimental)](HLSL_SM_6_8_WaveMatrix.md) documentation for more details about this feature, also called Wave Matrix Multiply and Accumulate or WaveMMA for short.
+See the [Work Graphs](WorkGraphs.md) documentation for more details.
```

Comment:
I think perhaps work graphs should be mentioned first as it is the highlight of this shader model.